1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,17 @@
-function setAlarm() {}
+function setAlarm() {
+  let inputEl = document.querySelector("#alarmSet");
+  let timer = inputEl.value;
+  let timeRemainEl = document.querySelector("#timeRemaining");
+  timeRemainEl.innerText = "Time Remaining: 00:" + timer;
```

Comment:
what happens here if the timer value is less than 10 seconds? or more than 59? I think some more formatting could be used of the input before it is displayed